libbeat/processors/dissect/processor_test.go

```diff
@@ -232,3 +232,179 @@ func TestErrorFlagging(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestIgnoreFailure(t *testing.T) {
```

Comment:
These lines are coming from https://github.com/elastic/beats/pull/19685. Maybe we should get that merged first (ping @adriansr) so this backport PR here becomes cleaner?

Reply:
ahh sorry I missed this before approving...

Reply:
woaw! Good catch! You really impressed me :smiley:

Reply:
Hmmm, now it seems like we are completely missing [`TestProcessorConvert`](https://github.com/elastic/beats/pull/18683/files?file-filters%5B%5D=.go#diff-e47381c4db222aafe9fba3a0526cf71cR414) in this backport?